Sources/SectionScrubber.swift

```diff
@@ -76,6 +98,10 @@ public class SectionScrubber: UIView {
             if let scrubberImage = self.scrubberImage {
                 self.scrubberWidth = scrubberImage.size.width
                 self.scrubberImageView.image = scrubberImage
+                // scrubber is hidden by default
```

Comment:
Comments should be used to explain why a particular piece of code does something.